src/apiCalls.js

```diff
@@ -19,4 +19,4 @@ const getSomeArt = (objectID) => {
     })
 }
 
-export default {findSomeArt, getSomeArt};
\ No newline at end of file
+export {findSomeArt, getThatArt};
```

Comment:
are you familiar with exporting functions when you create them? It can be a clean easy way to write it and that way you know you are exporting all the things needed. EX. 
export function getThatArt() {
...
}
then on the import side you just import whatever functions you need 
ex. 
import { getThatArt} from ....